package.json

```
@@ -1,12 +1,17 @@
 {
-  "name": "yandex-middle-dev",
+  "name": "my-webpack-project",
   "version": "1.0.0",
-  "description": "online six month course in yandex practicum",
+  "description": "My webpack project",
   "scripts": {
-    "start": "node src/server.js",
-    "build": "parcel build src/index.html --no-optimize",
-    "lint": "eslint src/**/**/*.ts src/*.js --fix",
-    "stylelint": "stylelint --fix \"src/sass/**/*.scss\" --custom-syntax postcss-scss"
+    "start": "node server.js",
+    "all": "webpack --mode=production --node-env=production",
+    "lint": "eslint src/**/**/*.ts --fix",
+    "stylelint": "stylelint --fix \"src/sass/**/*.scss\" --custom-syntax postcss-scss",
+    "build": "webpack --mode=development",
+    "build:prod": "webpack --mode=production --node-env=production",
+    "watch": "webpack --watch",
+    "serve": "webpack serve",
+    "test": "./node_modules/.bin/mocha"
```

Здесь нужно запускать стандартный скрипт вместо бинарника для того чтобы это работало на всех платформах
```suggestion
    "test": "mocha"
```